deploy/README.md

```diff
@@ -118,25 +118,9 @@ docker compose up --wait
 On first launch, WBS Deployment Kit will create files in the `./config` directory next to your `.env` file and the `docker-compose.yml` and `template.env`. This is the configuration of your instance. **You own those files.** Be sure to include them in your backups.
 
 #### `config/LocalSettings.php`
-This file is generated by the [MediaWiki installer script](https://www.mediawiki.org/wiki/Manual:Install.php) and augmented by the Wikibase containers `entrypoint.sh` on first launch. Once this file has been generated, you own it. This means you can make changes, you even might [need to make changes on major version updates](https://www.mediawiki.org/wiki/Manual:Upgrading#Adapt_your_LocalSettings.php). In order to ease keeping track of your own adjustments vs settings generated by the Wikibase container, we provide a file called `LocalSettings.override.php`. This file is the preferred location for your adjustments. See below.
+This file is generated by the [MediaWiki installer script](https://www.mediawiki.org/wiki/Manual:Install.php) and augmented by the Wikibase container's `entrypoint.sh` on first launch. Once this file has been generated, you own it, and it is the place to adjust your MediaWiki configuration to your needs. Note that you may [need to make changes in this file as part of major version updates](https://www.mediawiki.org/wiki/Manual:Upgrading#Adapt_your_LocalSettings.php).
 
-The absence of `LocalSettings.php` is the trigger for the Wikibase container to run the [MediaWiki installer script](https://www.mediawiki.org/wiki/Manual:Install.php). If you need to run the installer again, remove the generated `LocalSettings.php` file and restart your instance.
-
-#### `config/LocalSettings.override.php`
-This file is the location for your MediaWiki and Wikibase customizations. If you need to regenerate your `LocalSettings.php`, your `LocalSettings.override.php` is the safe place that is never rewritten, even when running the [MediaWiki installer script](https://www.mediawiki.org/wiki/Manual:Install.php).
-
-If you want to, for example, change your site name, just put the following in your `LocalSettings.override.php`.
-```php
-<?php
-$wgSitename = "My New Site Name";
```

Comment:
Can you see a place where this example could still live? I kinda liked it.

Reply:
For sure, will bring it back into that section.

Reply:
Actually, I'm going to leave it out for now. Now that `config/LocalSettings.php` is the prescribed place to make this change the user would actually be changing an existing entry for `$wgSiteName` for this vs overriding it. However, I can try and formulate another example in this section with a configurable that might be added to make things more concrete for the user.